### DISCUSSION TITLE
How to remove site name from the article titles?

### DISCUSSION THREAD

**Discussion by amitness** [mmistakes/minimal-mistakes#2603] [Q&A] [answered]
Hi @mmistakes,

I am looking for a way to remove the site name from the title. Currently, we can only control the separator from config, but I was wondering if there is some workaround to edit the article title format.

**Current Title**: Semi-Supervised Learning in Computer Vision **- Amit Chaudhary**
**What I want:** Semi-Supervised Learning in Computer Vision

**Comment by iBug** [comment, marked as answer]
Override [`_include/seo.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/seo.html) locally and modify this line:

<https://github.com/mmistakes/minimal-mistakes/blob/2ef1bfb85614602d849defa3c740e00e5817d84f/_includes/seo.html#L10>

You probably want to change it into

```liquid
{%- assign seo_title = page.title -%}
```

See also: <https://mmistakes.github.io/minimal-mistakes/docs/overriding-theme-defaults/>

**Comment by amitness** [comment]
@iBug Thank you. That worked.
